python/pyspark/mllib/clustering.py

```diff
@@ -562,5 +564,67 @@ def _test():
         exit(-1)
 
 
+class LDAModel(JavaModelWrapper):
```

Comment:
@davies In Scala, LDAModel is abstract.  LocalLDAModel and DistributedLDAModel inherit from it.  We should eventually have this same setup in Python.  What is needed to maintain backwards compatibility?  If we add this API in Spark 1.5, can we later make LDAModel abstract, and have LocalLDAModel and DistributedLDAModel inherit from it?